packages/contracts-bedrock/test/cannon/PreimageOracle.t.sol

```diff
@@ -1385,7 +1386,7 @@ contract PreimageOracle_LargePreimageProposals_Test is Test {
             LibKeccak.absorb(_stateMatrix, blockSlice);
             LibKeccak.permutation(_stateMatrix);
 
-            leaves_[i] = PreimageOracle.Leaf({
+            leaves_[i] = IPreimageOracle.Leaf({
```

Comment:
It looks like there's other `Leaf` usages where we still use `PreimageOracle` and not `IPreimageOracle` to reference them. Do we want to change those in this PR?

Reply:
Good point, will replace that.